build.gradle.kts

```diff
@@ -0,0 +1,35 @@
+import org.hypertrace.gradle.publishing.License.APACHE_2_0
+
+plugins {
+    `java-gradle-plugin`
+    id("org.hypertrace.ci-utils-plugin") version "0.1.1"
+    id("org.hypertrace.publish-plugin") version "0.3.1"
+}
+
+group = "org.hypertrace.gradle.code.style"
+
+java {
+    targetCompatibility = JavaVersion.VERSION_1_8
+    sourceCompatibility = JavaVersion.VERSION_1_8
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile("com.diffplug.spotless:spotless-plugin-gradle:5.11.0")
```

Comment:
Should we include this as a dependency, or let the clients include the dependency separately?